contact.html

```diff
@@ -22,10 +22,10 @@ <h2>Getting Help</h2>
     Include the version of the operating system and all relevant software, etc.
     in your report.</p>
 
-    <p><b>help.openstreetmap.org:</b> You can ask questions on <a
-    href="https://help.openstreetmap.org/">help.openstreetmap.org</a>.
+    <p><b>community.openstreetmap.org:</b> You can ask questions on <a
+    href="https://community.openstreetmap.org/">community.openstreetmap.org</a>.
     Please have a look around whether your question was answered already.
-    Many have been tagged with
+    Many have been answered at old help site and tagged with
```

Comment:
not sure is it a good idea to link these

maybe it should be removed?

(also, is it maybe "at **the** old help site"?)

Reply:
Good enough for now. We can probably remove the links to the help site a year or two down the line.